wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/linking/expression/values/LookupEvaluators.scala

```diff
@@ -112,6 +114,11 @@ object LookupEvaluators {
       case WomObject(_, _) => s"'Object'-type value did not contain the field '$key' at runtime".invalidNel
       case p: WomPair if key == "left" => p.left.validNel
       case p: WomPair if key == "right" => p.right.validNel
+      case WomMap(_, value) => Validated.fromOption(
+        o = value.collectFirst {
```

Comment:
Would there be any value in checking for duplicate keys here, or is there an earlier verification that renders them impossible?

Reply:
i.e. use `collect` instead of `collectFirst` and assert size is 1